test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -670,19 +670,21 @@ def init_servers(cls, init_server_func=None, skip=None):
         if init_server_func is None:
             init_server_func = cls.init_server
         if 'servers' in cls.conf:
+            server_param = cls.conf['servers']
             if 'comms' not in cls.conf and 'comm' not in cls.conf:
-                cls.conf['comms'] = cls.conf['servers']
-            if 'schedulers' not in cls.conf and 'scheduler' not in cls.conf:
-                cls.conf['schedulers'] = cls.conf['servers']
+                cls.conf['comms'] = server_param
+            if 'sched' not in cls.conf and 'scheds' not in cls.conf:
```

Comment:
Sorry @shilpakodli while discussion offline I overlooked self.sched, this should be 'scheduler' and 'schedulers'. can you please revert this naming?